src/plotting/task_create_scenario_comparison_tables.py

```diff
@@ -25,7 +25,7 @@ def _create_deps(scenarios, groupby):
     py_dependencies = {
         "config.py": SRC / "config.py",
         "scenario_config.py": SRC / "simulation" / "scenario_config.py",
-        "plotting.py": SRC / "plotting" / "plotting.py",
+        # "plotting.py": SRC / "plotting" / "plotting.py",
```

Comment:
We should delete this or comment it in; Same in other places

Reply:
Added them back in. It worked better to use `pytask.mark.persist` anyways.